python_modules/dagster-graphql/dagster_graphql_tests/graphql/__snapshots__/test_resources.ambr

```diff
@@ -5,6 +5,7 @@
       '__typename': 'Pipeline',
       'modes': list([
         dict({
+          'id': '32cf6d461cf1cab7270cea08d9c538e6095bd6be-default',
```

Comment:
contrast with 
`2b43566d7b3de2138b12c91b6953d6f1ddde0140-default` 
when `represented_pipeline.identifying_job_snapshot_id` is used

Reply:
i had to pull this from snapshot match test since the remote_repository_origin is not stable between local and buildkite